py5paisa/py5paisa.py

```diff
@@ -365,10 +365,14 @@ def place_order(self, **order):
         """
         Places a fresh order
         """
-        try:
-            if (order['Price'] >= 0 and order["ScripCode"] and order['Exchange'] and order['OrderType'] and order['Qty'] and order['ExchangeType']):
-                self.set_payload(order)
-                return self.order_request("OP")
+        order_list = ['ScripData','ScripCode']
+        try:
+            if (order['Price'] >= 0  and order['Exchange'] and order['OrderType'] and order['Qty'] and order['ExchangeType']):
```

Comment:
`order` accepts a number of keyword arguments, which will be packed into a dictionary,

why we would want to pass `ScripData` over `ScripCode` ? We should stick to convention 

You must be doing somewhere following ?
`place_order(ScripData=4242, ....)`
instead you should do
`place_order(ScripCode=4242, ....)`

If the py5apisa sdk is doing somewhere i.e. using `ScripData` over `ScipCode`  in `place_order` method you should fix that instead not this method, IMHO.

Reply:
We Support either of ScripData or ScripCode, its not ScripData over ScripCode
e.g. you can place trade either with ScripCode=1660 or ScripData=ITC_EQ